packages/core/src/style-shortcuts/transform.ts

```diff
@@ -0,0 +1,43 @@
+import { specialAliases, isInSpecialAliases, aliases } from './alias'
+
+function parseAttribute(attr: string) {
+  let dashIndex = attr.indexOf('-')
+  if (dashIndex < 0) {
+    return [attr, true] as const
+  } else {
+    let identifier = attr.slice(0, dashIndex)
+    if (identifier === 'min' || identifier === 'max') {
+      dashIndex = attr.indexOf('-', dashIndex + 1)
+      identifier = attr.slice(0, dashIndex)
+    }
+
+    return [identifier, attr.slice(dashIndex + 1)]
+  }
+}
+
+function normalizeValue(value: string | boolean | number) {
+  return !isNaN(value as unknown as number) && typeof value !== 'boolean'
```

Comment:
don't you want to cast it here too?

```suggestion
  return !isNaN(+value) && typeof value !== 'boolean'
```